client/download/update-overlay.tsx

```diff
@@ -127,38 +128,48 @@ export function UpdateDialog({
   readyToInstall,
   progress,
 }: UpdateDialogProps) {
-  const title = !hasDownloadError ? 'Update available' : 'Error downloading update'
-
+  const { t } = useTranslation()
+  const title = !hasDownloadError
+    ? t('clientUpdate.overlay.noErrorTitle', 'Update available')
+    : t('clientUpdate.overlay.errorTitle', 'Error downloading update')
   let content = <span />
   if (hasDownloadError) {
     content = (
       <Subtitle1>
-        There was an error downloading the update. Please restart and try again, or visit{' '}
-        <a href={makeServerUrl('/')} target='_blank' rel='noopener noreferrer'>
-          our website
-        </a>{' '}
-        to download the latest version.
+        <Trans t={t} i18nKey='clientUpdate.overlay.errorBody'>
+          There was an error downloading the update. Please restart and try again, or visit{' '}
+          <a href={makeServerUrl('/')} target='_blank' rel='noopener noreferrer'>
+            our website
+          </a>{' '}
+          to download the latest version.
+        </Trans>
       </Subtitle1>
     )
   } else if (readyToInstall) {
     content = (
       <Content>
         <Subtitle1>
-          A new update has been downloaded and is ready to install. Please restart the application
-          to continue.
+          {t(
+            'clientUpdate.overlay.newUpdateReady',
+            'A new update has been downloaded and is ready to install. ' +
+              'Please restart the application to continue.',
+          )}
         </Subtitle1>
         <RaisedButton
           onClick={() => ipcRenderer.send('updaterQuitAndInstall')}
-          label='Restart now'
+          label={t('clientUpdate.overlay.restartNowLabel', 'Restart now')}
```

Comment:
I don't really understand the reason for putting a `Label` suffix on things in this way, we don't put it on everything so the decision here seems very random.